### PR TITLE
[now-build-utils] Add types for SPRv2

### PR DESCRIPTION
**Pull request by leo**
This pull request adds two new types in preparation for Serverless Prerendering v2.

The first one is `Prerender`, which represents a pre-rendering bundle:

```
interface Prerender {
  expiration: number;
  lambda: Lambda;
  fallback: FileBlob | FileFsRef | FileRef;
}
```

The second one is `PrerenderGroup`, which defines a list of `Prerender` that should be invalidated at the same time – if one of them is invalidated:

```
interface PrerenderGroup {
  items: Array<Prerender>;
}
```

### CODE REVIEW

**Review by timneutkens — APPROVED** [review]

**Review by AndyBitz — APPROVED** [review]

File: packages/now-build-utils/src/prerender.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import FileRef from './file-ref';
+import { Lambda } from './lambda';
+
+interface PrerenderOptions {
+  expiration: number;
+  lambda: Lambda;
+  fallback: FileBlob | FileFsRef | FileRef;
+}
+
+export class Prerender {
```
Comment (AndyBitz) [inline packages/now-build-utils/src/prerender.ts:12]: should probably be export default

Comment (leo) [reply packages/now-build-utils/src/prerender.ts:12]: Great catch!

**Review by styfle — APPROVED** [review]

File: packages/now-build-utils/src/prerender.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
```
Comment (juancampa) [inline packages/now-build-utils/src/prerender.ts:12]: I was calling these `Prerederable`, not the best name either but at least it's a noun. Maybe `PrerenderFile`?

Comment (styfle) [reply packages/now-build-utils/src/prerender.ts:12]: 👍  to `Prerederable`

👎 to `PrerenderFile` since it doesn't implement the `File` interface

Comment (leo) [reply packages/now-build-utils/src/prerender.ts:12]: Personally (and @timneutkens), we see `Prerender` as a noun. So better, IMO.

**Pull request merged**